lib/msf/core/payload/windows/encrypted_payload_opts.rb

```diff
@@ -20,8 +20,7 @@ def initialize(info={})
       register_advanced_options(
       [
         OptBool.new('StripSymbols', [ false, 'Payload will be compiled without symbols', true ]),
-        OptString.new('OptLevel', [ false, 'The optimization level to compile with, e.g. O1, O2, O3, Os', 'O2' ]),
-        OptPath.new('LinkerScript', [ false, 'Linker script that orders payload functions', "#{LINK_SCRIPT_PATH}/func_order.ld" ]),
+        OptString.new('OptLevel', [ false, 'The optimization level to compile with, e.g. O0, O1, O2, O3, Os', 'O2', 'Og' ]),
```

Comment:
Since this is required to one of a list of specific values further down the stack, this should be an `OptEnum`.